README.rst

```diff
@@ -1,22 +1,69 @@
-FIWARE Policy Manager GE
-____________________
+================================
+FIWARE Policy Manager GE - Cloto
+================================
 
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
 
 
-Description
-===========
+This is the code repository for **FIWARE Policy Manager GE - Cloto**, a server that provides an API-REST to
+create rules associated to servers, subscribe servers to Context Broker to get information about resources
+consumption of that servers and launch actions described in rules when conditions are given.
 
-This module is part of FIWARE Policy Manager. It provides an API-REST to create rules associated to servers,
-subscribe servers to Context Broker to get information about resources consumption of that servers and launch actions
-described in rules when conditions are given.
+This project is part of FIWARE_ and `FIWARE Ops`_ suite of tools for the operation of FIWARE Lab.
```

Comment:
This project is part of FIWARE_. Check also the FIWARE Catalogue entry for Policy Manager <http://catalogue.fiware.org/enablers/policy-manager-bosun>__